package.json

```diff
@@ -26,7 +26,7 @@
     "lint": "eslint --ext .ts ./src ./test ./cypress",
     "lint.fix": "npm run lint -- --fix",
     "fetchContracts": "node scripts/fetchContracts.js",
-    "firebase": "npm run firebase-emulators:start",
+    "firebase": "cd firebaseFunctions && npm run build && cd ../ && npm run firebase-emulators:start",
```

Comment:
Wdyt about adding another script for building, similar to `firebase-functions:watch`

```ts
    "firebase-functions:build": "cd firebaseFunctions && npm run build",
```
Related: `firebase-e2e` will likely benefit from a fresh build as well.